sdk-exporter/prometheus/src/main/scala/org/typelevel/otel4s/sdk/exporter/prometheus/PrometheusTextRecord.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2024 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.otel4s.sdk.exporter.prometheus
+
+import cats.Show
+import cats.data.NonEmptyVector
+import cats.syntax.foldable._
+import cats.syntax.option._
+import cats.syntax.show._
+import org.typelevel.otel4s.sdk.exporter.prometheus.PrometheusTextRecord.PrometheusTextPoint
+
+import scala.collection.immutable.ListMap
+
+private[prometheus] final case class PrometheusTextRecord(
```

Comment:
```suggestion
private[prometheus] final case class PrometheusTextRecord(
private final case class PrometheusTextRecord(
```

In this case, it's the equivalent because `PrometheusTextRecord` is in the same package (prometheus).

Reply:
Well noted - adjusted.